JotunnLib/Managers/ZoneManager.cs

```diff
@@ -259,7 +259,14 @@ public bool AddCustomLocation(CustomLocation customLocation)
                 return false;
             }
 
-            customLocation.Prefab.transform.SetParent(LocationContainer.transform);
+            customLocation.Prefab.transform.SetParent(LocationContainer.transform); 
+            foreach(var zNetView in customLocation.ZoneLocation.m_netViews)
+            { 
+                if (!PrefabManager.Instance.Prefabs.ContainsKey(zNetView.gameObject.name.GetStableHashCode()))
```

Comment:
why is it done manually and not via PrefabManager.AddPrefab? that would give out a warning when the name is redundant. or is it common that prefabs are redundant in a zone?

Reply:
Yeah, it's likely a normal prefab that is already registered, only in rare cases would this actually be needed, if you add a custom prefab only in a location, but don't register it yourself (like an altar you can't build or something)
Most prefabs in the location will be normal things like walls & floors, which are already registered